public/v4/apps/chaskiq.yml

```diff
@@ -0,0 +1,156 @@
+          description: This is the admin password. Change it if you will.
+          defaultValue: $$cap_gen_random_hex(12)
+          label: Admin Password
+          validRegex: /^[^\@]{12,}$/
+        - id: $$cap_chaskiq_default_sender_email
+          label: Default sender email
+          description: 'Default email sender, like: notifications@mail.yourapp.com'
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_chaskiq_default_ongoing_domain
+          label: Default deliver domain
+          description: For multiple email deliver tasks, like mail.yourdomain.com
+        - id: $$cap_chaskiq_secret_key_base
+          defaultValue: $$cap_gen_random_hex(64)
+          label: Chaskiq Secret Key Base
+          description: The randomized string which is used to verify the integrity of signed cookies. Please use a string with more than 26 characters
+          validRegex: /^[^\@]{26,}$/
+        - id: $$cap_chaskiq_postgres_password
+          defaultValue: $$cap_gen_random_hex(12)
+          label: Postgres Password
+          description: Password must be at least 12 characters.  Please use a random string.
+          validRegex: /^[^\@]{12,}$/
+        - id: $$cap_chaskiq_redis_password
+          defaultValue: $$cap_gen_random_hex(12)
+          label: Redis Password
+          description: Password must be at least 12 characters. Please use a random string.
+          validRegex: /^[^\@]{12,}$/
+        - id: $$cap_chaskiq_aws_key_id
+          label: AWS access key id
+          description: Amazon access key id (for file storage and email dispatching S3 / SES / SNS)
+        - id: $$cap_chaskiq_aws_access_key
+          label: AWS access key
+          description: Amazon access key (for file storage and email dispatching S3 / SES / SNS)
+        - id: $$cap_chaskiq_aws_s3_bucket
+          label: AWS S3 bucket
+          description: Amazon S3 bucket name
+        - id: $$cap_chaskiq_aws_s3_region
+          label: AWS bucket region
+          description: 'AWS S3 bucket region like: us-east-1'
+        - id: $$cap_chaskiq_aws_sns_configuration_set
+          label: SNS configuration set
+          description: This is important for chaskiq to receive emails (optional)
+        - id: $$cap_chaskiq_scoutapp_key
+          label: Scout app key
+          description: Scout AMP key, for performance & error monitoring (optional)
+        - id: $$cap_chaskiq_bugsnag_key
+          label: Bugsnag API key
+          description: Bugsnag API key for error reporting (optional)
+        - id: $$cap_chaskiq_geocoder_service
+          label: Geocoder service name, like (maxmind, ip_info, baidu, yandex)
+          description: The complete list of services can be found at https://github.com/alexreisner/geocoder/tree/master/lib/geocoder/lookups
+        - id: $$cap_chaskiq_geocoder_api_key
+          label: Geocoder API key
+
+    instructions:
+        start: A full featured Live Chat, Support & Marketing platform
```

Comment:
Add some more details here, perhaps also mention "https://chaskiq.io" so that people know where to look for more info